contrib/dex/README.md

```diff
@@ -0,0 +1,65 @@
+## Exposing DEX service
+After the cluster is up, you have to manually expose dex service using a ELB or Ingress.
```

Comment:
@camilb ok 👍 Then, this PR is LGTM. Please remove the `WIP` in the title of this PR when you've done/this PR is ready to be merged.

I also prefer using ACM for managing the cert for dex ELB.
I have not yet tried it myself but [the ACM certificate resource](http://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-resource-certificatemanager-certificate.html) seems to allow us to create an ACM cert managed by a cfn stack. AFAICS, a manual certificate approval seems to be required by a human though(=it isn't fully automated)